corgidrp/walker.py

```diff
@@ -462,75 +460,120 @@ def run_recipe(recipe, save_recipe_file=True):
         with open(recipe_filepath, "w") as json_file:
             json.dump(recipe, json_file, indent=4)
 
+    # determine if this is a RAM-heavy recipe which needs crop-stack processing
+    #sort_pupilimg_frames included here b/c it sorts through a large number of frame (>700) b/c 
+    # EM gain cal files (sorted here and excluded, processed by SSC) are included in the visits
+    ram_heavy_bool = (recipe['name'] == 'noisemap_generation_2' or 
```

Comment:
Instead of having this big branching logic, let's just add a new field in the recipe .json file that's like "ram_heavy", and a pipeline parameter for ram_increment that defaults to 200, but can be changed (like how track_indiviudal_errors works). We can also make an issue and address this after this release.